src/constants/Options.js

```diff
@@ -0,0 +1,32 @@
+const Options = {
+  mode: "run",
+  // Run API
+  files: null,
+  version: "latest",
+  flavor: "sdk",
+  cacheDir: "./cache",
+  platforms: [],
+  currentPlatform: null,
+  downloadUrl: "https://dl.nwjs.io/",
+  manifestUrl: "https://nwjs.io/versions.json",
+  // Build API
+  appName: false,
+  appVersion: false,
+  buildDir: "./build",
+  buildType: "default",
+  forceDownload: false,
+  macCredits: false,
+  macIcns: false,
+  macZip: null,
+  macPlist: false,
+  winVersionString: {},
+  winIco: null,
+  useRcedit: false,
+  argv: [],
+  // Package API
+  zip: null,
+  zipOptions: null,
+  mergeZip: true,
+};
+
+export default Options;
```

Comment:
Add empty return to end of file to make git happy
```suggestion
export default Options;

```